lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -15,11 +17,24 @@ local weaponWeights = {
     m9k_m202          = 3,
     m9k_rpg7          = 3,
     m9k_minigun       = 6,
-    cw_m249_official = 3,
+    cw_m249_official  = 3,
     weapon_lfsmissilelauncher = 4,
 }
 pvpMoveSpeed = {}
 
+
+local plyMeta = FindMetaTable( "Player" )
+pvpMoveSpeed.wrappedFuncs = {
+    Player = {
+        SetRunSpeed = plyMeta.SetRunSpeed,
+        SetWalkSpeed = plyMeta.SetWalkSpeed,
+    }
+}
+
+local plyWraps = pvpMoveSpeed.wrappedFuncs.Player
+local slowWalkSpeed = 100 -- Default speed when holding +walk
```

Comment:
Nit:

Could define this up with the other default gmod speeds

Reply:
It's not up there since it's not a config thing.